docs/commands/Mock.mdx

```diff
@@ -336,11 +335,8 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 [https://pester.dev/docs/usage/mocking
 
 
- Mock](https://pester.dev/docs/usage/mocking
-
-
- Mock)
+ Mock]()
```

Comment:
This is interesting. Probably caused by this single line comment after closing the comment-block.

```
#>
    # Mock
    [CmdletBinding()]
    param( 
```

Reply:
Check, will also investigate